doc/application_workflow/webserving.md

```diff
@@ -0,0 +1,137 @@
+
+Step I: 
+Download the original installation files from https://github.com/ParsaLab/cloudsuite/tree/master/benchmarks/web-serving/db_server
+
+Step II:
+In the Dockerfile, comment out the following lines:
+```
+ENV web_host web_server
+RUN sed -i -e"s/HOST_IP/${web_host}:8080/" /elgg_db.dump
+CMD bash -c "/execute.sh ${root_password}"
+```
+
+Step III:
+Replace “files/execute.sh” with the following:
+
+```
+#!/bin/bash
+set -x
+service mysql restart
+# Wait for mysql to come up
+while :; do mysql -uroot -p${root_password} -e "status" && break; sleep 1; done
+
+mysql -uroot -p$root_password -e "create database ELGG_DB;"
+bash -c 'sed -i -e"s/HOST_IP/${web_host}:8080/" /elgg_db.dump'
+cat /elgg_db.dump | grep 8080
+
+# Need bash -c for redirection
+bash -c "mysql -uroot -p$root_password ELGG_DB < /elgg_db.dump"
+
+mysql -uroot -p$root_password -e "GRANT ALL PRIVILEGES ON *.* TO 'root'@'%' IDENTIFIED BY '$root_password' WITH GRANT OPTION; FLUSH PRIVILEGES;"
+
+service mysql stop 
+/usr/sbin/mysqld
+```
+
+Step IV: Same as Step IV when creating the docker image for the Web server.
+
+### Compose File for vSphere Integrated Containers Engine
+```
+version: '2'
+
+networks:
+  my_net:
+    driver: bridge
+
+services:
+  web_server:
+    #image: your_own_web_image
+    image: victest/web_elgg
+    container_name: web_server
+    networks:
+       - my_net
+    ports:
+      - "8080:8080"
+      - "25:25"
+
+  mysql_server:
+    #image: your_own_db_image
+    image: victest/web_db
+    container_name: mysql_server
+    command: [bash, -c, "/execute.sh"]
+    networks:
+       - my_net
+    environment:
+       - web_host=192.168.60.130 # This is the VCH_IP
+       - root_password=root  # password for the root user
+       
+   memcache_server:
+     image: cloudsuite/web-serving:memcached_server
+     container_name: memcache_server
+     networks:
+       - my_net    
+```
+
+### Deploy to Your VCH
+
+Once you already have a VCH deployed by vic-machine, go to the folder where you have the above “docker-compose.yml” file and execute the following command to start the Web-serving application:
+```
+$> docker-compose -H VCH_IP:VCH_PORT up –d
+```
+Here VCH_IP and VCH_PORT can be found from the standard output when you use “vic-machine-create” to launch the VCH. Now we are ready to view the website. Open a browser and navigate to http://web_server:8080. You should be able to see the following page:
```

Comment:
"vic-machine create", no hyphen.

navigate to http://VCH_IP:8080.  We make sure to use the IP address of the VCH as the IP of our webserver here.